### DISCUSSION TITLE
shinylive to windows webserver

### DISCUSSION THREAD

**Discussion by serrulate** [posit-dev/py-shiny#1471] [General]
Anyone had any luck publishing a quarto shinylive document to a windows webserver?   i worked through these directions 
https://github.com/wch/retirement-simulation-dashboard/?tab=readme-ov-file

however when i upload the _site file and navigate to index.html  i'm getting the following errors in the console of my web browser 
wasm instantiation failed!  pyodide-worker.js:2790

**Comment by jcheng5** [comment]
Do you have a link you can send us? Or if not, can you paste the entirety of your web browser console?

**Comment by serrulate** [comment]
unfortunately the webserver sits behind our firewall but attached is the image of the error,
maybe its a CORS issue? also attached is an error from the network tab.
![network](https://github.com/posit-dev/py-shiny/assets/114961738/611e0ed1-16eb-4fa9-9777-82526750d568)
![console](https://github.com/posit-dev/py-shiny/assets/114961738/b0ad85a5-170e-47e0-9085-cc75768e3920)
